rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -96,15 +120,27 @@ add_dependencies(
 
 if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
   if(NOT "${_generated_msg_files} " STREQUAL " ")
+    _ament_cmake_python_get_python_install_dir()
```

Comment:
I know. The comment below explains that I couldn't get `ament_python_install_package` to work properly, so I put this up for review to ask for feedback. I'd rather use `ament_python_install_package` instead of relying on this private function.